src/translations/en.json

```diff
@@ -5566,6 +5566,17 @@
             "devices": "{count} {count, plural,\n  one {device}\n  other {devices}\n}",
             "provisioned_devices": "Provisioned devices",
             "not_ready": "{count} not ready",
+            "nvm_backup": {
+              "title": "Backup and Restore",
+              "description": "Back up or restore your Z-Wave controller's Non-Volatile Memory (NVM). The NVM contains your network information including paired devices and their configurations. It's recommended to create a backup before making any major changes to your Z-Wave network.",
```

Comment:
This is not entirely correct. The device configurations are stored on the devices. To keep it simple, I'd probably remove `and their configurations`.